.github/actions/publish-nuget/action.yml

```diff
@@ -124,25 +124,26 @@ runs:
       # Notes on the configuration apart from what's also for dotnet build:
       # * NoWarn on NU5104 to not have warnings for prerelease dependencies, see:
       #   https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104.
-      # * -p:WarnOnPackingNonPackableProject=True will cause a build warning (converted to error) if we try to pack
-      #   a non-packagable project.
+      # * -p:WarnOnPackingNonPackableProject=True will cause a build warning (converted to error) if we try to pack a
+      #   non-packagable project.
       # * -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg are needed to generate symbol packages:
       #   https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg.
+      # * p:EnablePackageValidation=true is for package validation, see:
+      #   https://docs.microsoft.com/en-us/dotnet/fundamentals/package-validation/overview.
       run: |
         if ([string]::IsNullOrWhiteSpace($Env:API_KEY)) {
           Write-Output "::error::API_KEY is missing or empty."
           exit 1
         }
 
-        $noWarn = @('NU5104') + @'
+        $noWarn = @('NU5104%3BCS1573%3BCS1591%3BVSTHRD002%3BVSTHRD200') + @'
```

Comment:
Why these new suppressions are required here?
I didn't find any information about this under the issue.

Reply:
These are the same as the build ones, because I removed `--no-build` from the `pack` command. That was necessary, since, apparently, `-p:EnablePackageValidation=true` invokes the "Build" target, see https://github.com/Lombiq/UI-Testing-Toolbox/runs/8234626733?check_suite_focus=true#step:19:140